systemtests/singlehost/sanity_test.go

```diff
@@ -194,3 +195,72 @@ func TestSingleHostMultiVlanPingFailure_sanity(t *testing.T) {
 		utils.DockerCleanup(t, node, "myContainer2")
 	}()
 }
+
+// Default Network Assignment and freeing
+func TestSingleHostDefaultNetwork_sanity(t *testing.T) {
```

Comment:
Sure, will add the test case for `ip route` based confirmation as seen by the container. On the other hand, for the removal of the network, few questions popped up:
- We do not implicitly delete the endpoints when a network is deleted this means we'll not remove the associated default route as well.

So, perhaps we should take the endpoint deletion upon network deletion separately. For this, I can ensure that endpoint deletion removes the default route.

-